src/components/chatlist/chat.js

```diff
@@ -193,98 +226,163 @@ import {
+            {!activeContact ? (
+                <Grid className={classes.rootList} container justify="center" alignItems="center" spacing={3}>
+                  <Grid item xs={6}>
+                    <h3>Start swiping and create matches first</h3>
+                  </Grid>
+                  <Grid item xs={3}>
+                    <Button
+                        variant="contained"
+                        className={classes.swipingButton}
+                        onClick={() => history.push('/swipe/' + idNumber)}
+                    >
+                      Start Swiping
+                    </Button>
+                  </Grid>
+                </Grid>
+            ):(
+                <div>
+                  <h5>Chat with {activeContact.name}</h5>
+                  <Divider />
+                  <List className={classes.rootList}
+                        component="nav"
+                  >
+                    {messages.map((msg, index) => {
+                      return (
+                          <div key={index}>
+                            <ListItem ref={scrollRef} >
+                              <ListItemText style={{display:'flex', justifyContent: msg.senderId == idNumber ? 'flex-end' : 'flex-start'}} primary={msg.content}/>
+                            </ListItem>
+                          </div>
+                      );
+                    })}
+                  </List>
                 </div>
-              );
-              })}
-            </List>
+            )}
           </Panel>
         </Grid>
+
         <Grid item xs={4}>
           <Panel shaded>
-            <div>your current item</div>
-            <div>Title: {curremtItem.title}</div>
-            <div>Description: {curremtItem.description}</div>
-            <Picture itemId={curremtItem.id}/>
-          </Panel>
-        </Grid>
-        <Grid item xs={6}>
-          <Panel shaded>
-            <TextField 
-              margin="normal"
-              required
-              fullWidth
-              placeholder="send text"
-              value={text}
-              autoFocus
-              onChange={(event) => setText(event.target.value)}
-              onKeyPress={(event) => {
-                if (contacts && event.key === "Enter") {
-                  sendMessage(text);
-                  setText("");
-                }
-              }}
-            />
-            <Button
-              variant="contained"
-              onClick={() => {
-                if(contacts) {
-                  sendMessage(text);
-                  setText("");
-                }
-              }}
-            >
-              Send
-            </Button>
-            <Game stomp={stompClient} id={idNumber} activeContact={stateRef.current} curremtItem={curremtItem} sendMessage={sendMessage}></Game>
+            <Grid container justify="flex-start" alignItems="center">
+              <Grid item xs={4}>
+                <Picture itemId={currentItem.id}/>
```

Comment:
somthing like {curentItem.id && <Picture itemId={currentItem.id}/>} to make the picture work in the chat